### DISCUSSION TITLE
Any plan to support Hot Reload on Android?

### DISCUSSION THREAD

**Discussion by jojoclt** [JetBrains/compose-hot-reload#57] [General]
I have tried to set this up, whenbusing the Firework Compiler, it says starting from Kotlin 2.0.0, I need to enable the Compose Compiler Plugin

**Comment by ferdiallen** [comment]
Same here, it would greatly increase my development production because with targeting android, i can easily test my adaptive layout for mobile devices

**Comment by Kashif-E** [comment]
issue++

**Comment by sellmair** [comment]
Android has 'Live Edit' which should provide a similar user experience when targeting Android directly.
We're in contact with the Android (and Live Edit) team and proposed to merge efforts in some areas. 
Please stand by for further updates.
